packages/e2e/cypress/e2e/app/dates.cy.ts

```diff
@@ -126,144 +159,169 @@ describe('Date tests', () => {
+    it('Should use UTC dates', () => {
+        const exploreStateUrlParams = `?create_saved_chart_version={"tableName"%3A"events"%2C"metricQuery"%3A{"exploreName"%3A""%2C"dimensions"%3A["events_timestamp_tz_raw"]%2C"metrics"%3A["events_count"]%2C"filters"%3A{"dimensions"%3A{"id"%3A"3b565490-87c5-4996-a42b-ff0640bb18cd"%2C"and"%3A[{"id"%3A"be863f3c-5807-48c5-9b6f-2e8445610280"%2C"target"%3A{"fieldId"%3A"events_timestamp_tz_raw"}%2C"operator"%3A"equals"%2C"values"%3A["2020-08-12T00%3A58%3A00%2B02%3A00"]}]}}%2C"sorts"%3A[{"fieldId"%3A"events_timestamp_tz_raw"%2C"descending"%3Atrue}]%2C"limit"%3A500%2C"tableCalculations"%3A[]%2C"additionalMetrics"%3A[]}%2C"tableConfig"%3A{"columnOrder"%3A["events_timestamp_tz_raw"%2C"events_count"]}%2C"chartConfig"%3A{"type"%3A"cartesian"%2C"config"%3A{"layout"%3A{"xField"%3A"events_timestamp_tz_raw"%2C"yField"%3A["events_count"]}%2C"eChartsConfig"%3A{"series"%3A[{"type"%3A"bar"%2C"yAxisIndex"%3A0%2C"encode"%3A{"xRef"%3A{"field"%3A"events_timestamp_tz_raw"}%2C"yRef"%3A{"field"%3A"events_count"}}}]}}}}`;
         cy.visit(
-            `/projects/${SEED_PROJECT.project_uuid}/tables/orders${exploreStateUrlParams}`,
+            `/projects/${SEED_PROJECT.project_uuid}/tables/events${exploreStateUrlParams}`,
         );
+        cy.contains('SQL');
+        cy.findAllByText('Loading chart').should('have.length', 0);
 
+        cy.contains('1 active filter');
         cy.findByTestId('Filters-card-expand').click();
-        cy.findAllByText('Loading chart').should('have.length', 0);
-        cy.findByDisplayValue('2022-07-11, 14:23:11:000');
-        cy.get('svg g text').contains('2017-02-11, 03:00:00:000 (+00:00)');
-        cy.get('tbody td').contains('2017-02-11, 03:00:00:000 (+00:00)');
+        cy.contains('11 Aug 2020 22:58:00'); // Filter in UTC
+
+        cy.contains(`2020-08-11, 22:58:00:000 (+00:00)`); // Data in results, this comes from the server, so depends on the server timezone
```

Comment:
This is very interesting, the +00:00 can change depending on the server timezone, so if I'm running `backend` on UTC+2, this will show +02:00, it doens't matter what's my warehouse